src/coreclr/jit/compiler.cpp

```diff
@@ -5359,11 +5359,11 @@ PhaseStatus Compiler::placeLoopAlignInstructions()
                 madeChanges       = true;
                 unmarkedLoopAlign = true;
             }
-            else if (visitedLoopNum[loopTop->bbNatLoopNum])
+            else if ((loopTop->bbNatLoopNum != BasicBlock::NOT_IN_LOOP) && (visitedLoopNum[loopTop->bbNatLoopNum]))
```

Comment:
Don't need so many parens
```suggestion
            else if ((loopTop->bbNatLoopNum != BasicBlock::NOT_IN_LOOP) && visitedLoopNum[loopTop->bbNatLoopNum])
```